src/transformers/configuration_utils.py

```diff
@@ -249,6 +247,11 @@ def __init__(self, **kwargs):
 
         # TPU arguments
         self.xla_device = kwargs.pop("xla_device", None)
+        if kwargs.pop("xla_device", None) is not None:
+            logger.warn(
+                "The `xla_device` argument has been deprecated in v4.4.0 of Transformers. It is ignored and you can "
+                "safely remove it from your `config.json` file."
```

Comment:
Could this log inform users how to proceed now, or if they need to do anything?

Reply:
They don't need to do anything, apart from removing the arg from their `config.json` if they want the warning to disappear.